modules/images-imagestream-external-image-tags.adoc

```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
 
 * Add tags pointing to internal or external images, by using the `oc tag` command
  for all tag-related operations:
- +
++
 ----
-oc tag <repositiory/image> <image-name:tag>
+oc tag <image-name:tag> <image-name:tag>
```

Comment:
The cmd here 'oc tag <repositiory/image> <image-name:tag>'  is correct, the purpose is tagging an external image.